pkgs/applications/science/programming/scyther/sources.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub }:
+let
+  versions = map (s: { name = s.rev; } // s) [
```

Comment:
Heh, yes that is a very valid point, I thought it would be neat if it could be chosen between two releases which very much differ (very comparable to firefox and firefox-esr).

How about this: Getting rid of the `master` version (this can also fix the rev as version in your other comment) and making it possible to override source like all other packages?